kubernetes/resource_kubernetes_annotations.go

```diff
@@ -261,7 +267,7 @@ func resourceKubernetesAnnotationsUpdate(ctx context.Context, d *schema.Resource
 		types.ApplyPatchType,
 		patchbytes,
 		v1.PatchOptions{
-			FieldManager: defaultFieldManagerName,
+			FieldManager: d.Get("field_manager").(string),
```

Comment:
This might actually get set to empty string by the user.
A quick test with empty `field_manager` yields this error:
```
Error: PatchOptions.meta.k8s.io "" is invalid: fieldManager: Required value: is required for apply patch
```
We could consider only setting this when it has a meaningful value.

Reply:
I've added a validator so the user can't set this value to empty